libs/libjson/_internal/Source/JSONNode.h

```diff
@@ -161,7 +161,7 @@ class JSONNode {
 
 
     json_string as_string(void) const json_nothrow json_read_priority;
-    int as_int(void) const json_nothrow json_read_priority;
```

Comment:
I think applying changes to json is tricky move. It is an external library and the day X we will want to update it from its main development repo. Then with the update we will loose any changes we applied. I would recommend to add a patch file with these changes, and specify the revision of the original json repo.